_includes/releases/v22.2/v22.2.8.md

```diff
@@ -7,46 +7,46 @@ Release Date: April 17, 2023
-- Fixed a rare bug existing since before v22.1 that could cause a projected expression to replace column references with the wrong values. [#97593][#97593]
-- Fixed a bug where concurrent DML queries could corrupt an existing primary index when [adding a new column](../v22.2/alter-table.html) with a new column family. If a rollback occurs the table may no longer be accessible. [#100187][#100187]
+- Fixed a rare bug introduced before v22.1 that could cause a projected expression to replace column references with incorrect values. [#97593][#97593]
+- Fixed a bug in the new [declarative schema changer](https://www.cockroachlabs.com/docs/stable/online-schema-changes.html#declarative-schema-changer) that could cause primary index corruption when an `ALTER TABLE..ADD COLUMN` statement executed concurrently with an `UPDATE` or `INSERT` statement if the schema change failed and was rolled back. [#100187][#100187]
 
 <h3 id="v22-2-8-performance-improvements">Performance improvements</h3>
 
-- Removed prettify usages that could cause OOM on [Statements](../v22.2/ui-statements-page.html) and [Transactions](../v22.2/ui-transactions-page.html) page. [#99453][#99453]
-- Audit logging could incur extra latency when resolving table/view/sequence names. To address this, this patch adds support for using the lease cache for these looks up and eliminate any extra round trips. [#99661][#99661]
-- The execution of multiple [`FOREIGN KEY`](../v22.2/foreign-key.html) and [`UNIQUE`](../v22.2/unique.html) constraint checks can be parallelized in some cases. As a result, these checks can be completed faster, especially so in multi-region environments where the checks require cross-region reads. The feature is enabled by default on v23.1 release, and in order to enable it on v22.2, one must change the private `sql.distsql.parallelize_checks.enabled` [cluster setting](../v22.2/cluster-settings.html) to `true`. [#100520][#100520]
+- Removed prettify usages that could cause out-of-memory (OOM) errors on the [Statements](../v22.2/ui-statements-page.html) and [Transactions](../v22.2/ui-transactions-page.html) pages. [#99453][#99453]
+- Audit logging now looks up the names of tables, views, and sequences in a node's lease cache to reduce network latency. [#99661][#99661]
+- The execution of multiple [`FOREIGN KEY`](../v22.2/foreign-key.html) and [`UNIQUE`](../v22.2/unique.html) constraint checks can be parallelized in some cases to help them complete more quickly. faster, especially so in multi-region environments where the checks require cross-region reads. This feature is disabled by default. To enable it, set the private (undocumented) `sql.distsql.parallelize_checks.enabled` [cluster setting](../v22.2/cluster-settings.html) to `true`. [#100520][#100520]
```

Comment:
Are we OK with having this in a release note or should this be support-only since it is private?